feature/platform/tests/power_admin_down_up_test/power_admin_down_up_test.go

```diff
@@ -26,13 +26,18 @@ func TestFabricPowerAdmin(t *testing.T) {
 
 	for _, f := range fs {
 		t.Run(f, func(t *testing.T) {
+
+			if !gnmi.Get(t, dut, gnmi.OC().Component(f).Removable().State()) {
+				t.Skipf("Skip the test on non-removable fabric.")
+			}
+
 			oper := gnmi.Get(t, dut, gnmi.OC().Component(f).OperStatus().State())
 
 			if got, want := oper, oc.PlatformTypes_COMPONENT_OPER_STATUS_ACTIVE; got != want {
 				t.Skipf("Fabric Component %s is already INACTIVE, hence skipping", f)
 			}
 
-			before := helpers.FetchOperStatusUPIntfs(t, dut, false)
+			before := helpers.FetchOperStatusUPIntfs(t, dut, true)
```

Comment:
Please check all interfaces instead of just the ones in the binding.

Reply:
Done. Thanks.